docs/public/specifications/tests/DNSSEC-TP/dnssec10.md

```diff
@@ -53,37 +56,45 @@ This test case is only relevant if the zone has been DNSSEC signed.
+DS10_NSEC3_ERR_TYPE_LIST           | ERROR   | ns_ip_list | NSEC3 record for the zone apex with incorrect type list. Fetched from nameservers with IP addresses "{ns_ip_list}".
+DS10_NSEC3_MISMATCH_APEX           | ERROR   | ns_ip_list | NSEC3 record with a non-apex owner name, which is against the expectation. Fetched from nameservers with IP addresses "{ns_ip_list}".
+DS10_NSEC3_MISSING_SIGNATURE       | ERROR   | ns_ip_list | Missing signatures (RRSIG) for the NSEC3 record or records. Fetched from the nameservers with IP addresses "{ns_ip_list}".
+DS10_NSEC3_NODATA_MISSING_SOA      | ERROR   | ns_ip_list | Missing SOA record in NODATA response with NSEC3. Fetched from nameservers with IP addresses "{ns_ip_list}".
+DS10_NSEC3_NODATA_WRONG_SOA        | ERROR   | ns_ip_list | Wrong owner name on SOA record in NODATA response with NSEC3. Fetched from nameservers with IP addresses "{ns_ip_list}".
+DS10_NSEC3_RRSIG_VERIFY_ERROR      | ERROR   |ns_ip_list, keytag| The signatures (RRSIG with tag {keytag}) for the NSEC3 record cannot be verified. Fetched from the nameservers with IP addresses "{ns_ip_list}".
+DS10_NSEC_ERR_TYPE_LIST            | ERROR   | ns_ip_list | NSEC record for the zone apex with incorrect type list. Fetched from nameservers with IP addresses "{ns_ip_list}".
+DS10_NSEC_MISMATCH_APEX            | ERROR   | ns_ip_list | NSEC record with a non-apex owner name, which is against the expectation. Fetched from nameservers with IP addresses "{ns_ip_list}".
+DS10_NSEC_MISSING_SIGNATURE        | ERROR   | ns_ip_list | Missing signatures (RRSIG) for the NSEC record or records. Fetched from the nameservers with IP addresses "{ns_ip_list}".
+DS10_NSEC_NODATA_MISSING_SOA       | ERROR   | ns_ip_list | Missing SOA record in NODATA response with NSEC. Fetched from nameservers with IP addresses "{ns_ip_list}".
+DS10_NSEC_NODATA_WRONG_SOA         | ERROR   | ns_ip_list | Wrong owner name on SOA record in NODATA response with NSEC. Fetched from nameservers with IP addresses "{ns_ip_list}".
+DS10_NSEC_QUERY_GIVES_ERR_ANSWER   | ERROR   | ns_ip_list | Unexpected DNS record in the answer section on an NSEC query. Fetched from nameservers with IP addresses "{ns_ip_list}".
+DS10_NSEC_QUERY_RESPONSE_ERROR     | ERROR   | ns_ip_list | No response or error in response on query for NSEC. Fetched from the nameservers with IP addresses "{ns_ip_list}".
+DS10_NSEC_RRSIG_VERIFY_ERROR       | ERROR   | ns_ip_list | The signatures (RRSIG with tag {keytag}) for the NSEC record cannot be verified. Fetched from the nameservers with IP addresses "{ns_ip_list}".
+DS10_SERVER_NO_DNSSEC_SUPPORT      | ERROR   | ns_ip_list | The name servers listed here do not support DNSSEC or has not properly configured. Testing for NSEC and NSEC3 has been skipped on these servers. Fetched from the nameservers with IP addresses "{ns_ip_list}".
```

Comment:
MSIG updated.

I have changed to `ns_list`. Why here and not in every place?

Reply:
In general, I think `ns` and `ns_list` to be more helpful in the output. They require a little more specification if we are strict. Usually we want to loop over the IP addresses, and then what should be reported if the there are two names to the same IP address? Just report one of them (kind of randomly)?

How much extra will the implementation require to output `ns` (`ns_list`) instead of `ns_ip` (`ns_ip_list`)?

One decision could be to say that in the specification the name server name could be implicit except for that in the `msgid` it must be spelled out if it should be `ns` (`ns_list`) or `ns_ip` (`ns_ip_list`).

Reply:
@tgreenx, would it then be OK to say that in the specification it normally only refer to the name server IP except in the msgid where it should be explicit if it should be `{ns}` or `{ns_ip}`? If we think that would be good, then that could be spelled out in the template.

Reply:
Let us try that. See updates.